components/console_cmd_ping/idf_component.yml

```diff
@@ -0,0 +1,6 @@
+version: 0.0.3
+url: https://github.com/espressif-abhikroy/esp-protocols/tree/components/console_cmd_ifconfig/components/console_cmd_ping
+description: The component provides a console where the 'ping' command can be executed.
+dependencies:
+  idf:
+    version: '>=5.1'
```

Comment:
Is there anything in this component that doesn't work for v5.0? If no, could we relax the version requirement to allow v5.0 as well?

Reply:
Both console_cmd_ifconfig and console_cmd_ping now works with v5.0.